curvine-fuse/src/fs/curvine_file_system.rs

```diff
@@ -656,11 +656,7 @@ impl fs::FileSystem for CurvineFileSystem {
             | "security.selinux"
             | "system.posix_acl_access"
             | "system.posix_acl_default" => {
-                // Silently ignore system extended attributes without logging
-                // Return ENODATA error to indicate the attribute doesn't exist
-                // This is the correct FUSE protocol response for non-existent attributes
-                // Empty error message to avoid any logging overhead
-                return err_fuse!(libc::EOPNOTSUPP, "not support get_xattr {}", name);
+                return err_fuse!(libc::ENODATA, "get_xattr {}", name);
```

Comment:
According to the POSIX specification, EOPNOTSUPP should be returned here.

Reply:
No, If the kernel send the getxattr request, should return ENODATA if we don't support. otherwise the kernel will return operatioin not support error.

Reply:
> The disk file system returns an EOPNOTSUPP error for these attribute pairs.

If return ENOSUPPORT, when lookup nobody:nobody dir, the filesystem will report Operation not support error, it is not ok

Reply:
Yes, you're right.